ui/apps/dashboard/src/components/Billing/Addons/EntitlementListItem.tsx

```diff
@@ -0,0 +1,137 @@
+'use client';
+
+import { Button } from '@inngest/components/Button';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@inngest/components/Tooltip/Tooltip';
+import { RiInformationLine } from '@remixicon/react';
+
+import EntitlementListItemSelfService from '@/components/Billing/Addons/EntitlementListItemSelfService';
+import { addonPriceStr } from '@/components/Billing/Addons/pricing_help';
+import { pathCreator } from '@/utils/urls';
+
+export default function EntitlementListItem({
+  title,
+  description,
+  tooltipContent,
+  selfServiceAvailable,
+  canIncreaseLimitInCurrentPlan,
+  entitlement,
+  addon,
+  onChange,
+}: {
+  title: string; // Title of the addon (e.g. Users, Concurrency)
+  description?: string;
+  tooltipContent?: string | React.ReactNode;
+  selfServiceAvailable: boolean; // Is an addon available (for this account) via self-service or will they need to contact us?
+  canIncreaseLimitInCurrentPlan: boolean; // Can this limit be increased on the current billing plan (or does the user need to upgrade to a higher plan)?
```

Comment:
Can this be inferred by checking if `addon` is truthy? If an entitlement has an addon then an increase can be purchased within the plan, else an upgrade is necessary to increase the entitlement